examples/json-io/json_io.bal

```diff
@@ -13,7 +13,7 @@ public function main() returns @tainted error? {
             "codes": ["4", "8"]
         }};
 
-    // Write the given JSON to a file.
+    // Writes the given JSON to a file.
     check io:fileWriteJson(jsonFilePath, jsonContent);
     // If the write operation was successful, then, perform a read operation to read the JSON content.
```

Comment:
```suggestion
    // If the write operation was successful, then, performs a read operation to read the JSON content.
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/e7dc5ab61d2e03737d6158d4fd59a02371e55d84